data/infra/resources/remote_file.yaml

```
@@ -9,7 +9,7 @@ resource_description_list:
 - markdown: Use the **remote_file** resource to transfer a file from a remote location
     using file specificity. This resource is similar to the **file** resource.
 - note:
-    markdown: Fetching files from the `files/` directory in a cookbook should be done
+  markdown: Fetching files from the `files/` directory in a cookbook should be done
```

This time my suggestion should be right.
```suggestion
    markdown: Fetching files from the `files/` directory in a cookbook should be done
```